src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -150,8 +156,8 @@ class RobotContainer {
 
     JoystickButton(controller1, kB.value).whenActive(
             ParallelCommandGroup(
-                    FixedIndexerSpeed(indexer, {0.0}),
-                    FixedIntakeSpeed(intake, {0.0})
+                    FixedIndexerSpeed(indexer, { 0.0 }),
+                    FixedIntakeSpeed(intake, { -1.0 })
```

Comment:
This -1.0 should be pulling from constants in case we need to change the speed. Maybe `-1.0 * Constants.kIntakeSpeed`.